src/index.js

```diff
@@ -30,3 +30,4 @@ export {
   ScenarioValidationStatusChip,
   DefaultAvatar,
 } from './misc';
+export { getIentifierFromUserEmail } from './utils';
```

Comment:
You can remove this export from index.js, because don't want to export this function in the lib, otherwise we will have to maintain it and guarantee backward-compatibility.
This function is just an internal function that we want to use in several of our components :slightly_smiling_face: